### PR TITLE
C2 rev

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
@Nowosad good to merge I'd say - just put this in a PR to show it's a more-or-less cohesive review. If only all reviews came in the form of a PR! Any further changes you'd like to see before treating this chapter as 'done'? Feel free to add them to this branch or to `master` once this goes through.

**Comment by Nowosad** [comment]
@Robinlovelace I'll look at this later in the day.

File: 02-spatial-data.Rmd [hunk 1]
```diff
@@ -648,6 +649,54 @@ This section is work in progress.
...
 
+## Coordinate Reference Systems
+
+This section is in progress.
+
+Despite the differences between vector and raster spatial data types, they are united by shared concepts intrinsic to spatial data. Perhaps the most important of these is Coordinate Reference System (CRS), which defines how the spatial elements of the data relate to the surface of the Earth (or other body).
+
+In **sf** the CRS of an object can be retrieved and set using `st_crs()`:
+
+```{r}
```
Comment (Nowosad) [inline 02-spatial-data.Rmd]: You can also use `st_set_crs()`.

Comment (Robinlovelace) [inline 02-spatial-data.Rmd]: In there now.

**Pull request merged**